packages/admin/cms-admin/src/generator/future/generateForm.ts

```diff
@@ -7,6 +8,10 @@ import { findMutationTypeOrThrow } from "./utils/findMutationType";
 import { generateImportsCode, Imports } from "./utils/generateImportsCode";
 
 export type Prop = { type: string; optional: boolean; name: string };
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
+function isProp(arg: any): arg is Prop {
```

Comment:
Why not use unknown instead?

Reply:
I would like to use unknown, but it does not work for predicate. I could use `Prop | { [key: string]: ObjectType | Prop }`, but I don't think this would bring any benefit.

<img width="489" alt="image" src="https://github.com/user-attachments/assets/147ca3d0-22f5-4b84-a529-851fff71a724">

Reply:
You could use `typeof arg === "object" && arg !== null && "name" in arg && "type" in arg` 😁